src/index.html

```diff
@@ -6,25 +6,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
-    <title>List Structure</title>
+    <title>Todo List</title>
   </head>
   <body>
     <section class="form-section">
       <form id="form" class="form-control">
         <div class="header-wrap">
           <h3 class="heading">Today's To Do</h3>
-          <span class="syc"><i class="fas fa-sync"></i></span>
         </div>
         <div class="form-input">
           <input type="text" name="text" id="text" placeholder="Add to your list..." class="italics"/>
```

Comment:
- [ ] It would be nice if upon pressing enter after typing in a task, it gets aded to the list, unlike where currently the text disappers making a bad user experience
![image](https://user-images.githubusercontent.com/81932478/213201652-c8ea1d57-b05d-4855-9560-b94de0122ae2.png)